djangocms_alias/test_utils/text/templates/text/text.html

```diff
@@ -1 +1 @@
-{{ instance.body|safe }}
\ No newline at end of file
+{{ instance.body|safe }}
```

Comment:
Here, a newline has been added which must be removed. Sometimes editors do this. I have this issue with PyCharm, for example.

Reply:
Ah, there we go. I'll also have to do this for djanogcms-text[-ckeditor]...